watchtower-plugin/tests/test.py

```diff
@@ -156,3 +156,34 @@ def test_misbehaving_watchtower(node_factory, bitcoind, teosd, directory):
     l1.rpc.pay(l2.rpc.invoice(25000000, "lbl1", "desc1")["bolt11"])
     assert l2.rpc.gettowerinfo(tower_id)["status"] == "misbehaving"
     assert l2.rpc.gettowerinfo(tower_id)["misbehaving_proof"]
+
+
+def test_get_appointment(node_factory, bitcoind, teosd, directory):
+    l1, l2 = node_factory.line_graph(2, opts=[{}, {"plugin": "watchtower-client"}])
+
+    # We need to register l2 with the tower
+    tower_id = teosd.cli.get_tower_info()["tower_id"]
+    l2.rpc.registertower(tower_id)
+
+    # Force a new commitment
+    l1.rpc.pay(l2.rpc.invoice(25000000, "lbl1", "desc1")["bolt11"])
+    tx = l1.rpc.dev_sign_last_tx(l2.info["id"])["tx"]
+
+    # Now make sure it is out of date
+    l1.rpc.pay(l2.rpc.invoice(25000000, "lbl2", "desc2")["bolt11"])
+    # Add some waiting so the tower is not shutdown too early
+    time.sleep(2)
```

Comment:
Looks like adding a small sleep did the trick. Not ideal but I do like it better than allowing broken logs.